app/controllers/phrase_usage_controller.rb

```diff
@@ -0,0 +1,7 @@
+class PhraseUsageController < ApplicationController
+  def show
+    @phrase = Phrase.find(params[:id])
+    @survey_answers_containing_phrase = SurveyAnswer.find_by_sql(["select * from survey_answers sa join questions q on q.id = sa.question_id join survey_phrases sp on sp.survey_answer_id = sa.id join phrases p on p.id = sp.phrase_id where p.id = ? and q.question_number = 3 and sa.answer not like '-' limit 10", "#{params[:id]}"])
+    # @survey_answers_containing_phrase = Phrase.find_by_sql('select phrase.* from survey_answers sa join questions q on q.id = sa.question_id join survey_phrases sp on sp.survey_answer_id = sa.id join phrases p on p.id = sp.phrase_id where q.question_number = 3 and sa.answer not like \'-\' limit 10')
```

Comment:
If this isn't required/doesn't work I'd remove it, keep a local copy if you want to hang onto it

Reply:
Also meant to delete, thanks